Frontend/src/Components/FilteredListing/FilteredListing.jsx

```diff
@@ -27,21 +27,13 @@ const FilteredListings = () => {
   }, [filterType, filterValue]);
 
   return (
-    <div className="filteredListingsContainer">
-      <AppHeader />
-      <div className="filteredListingsGrid">
-        {listings.map((listing) => (
-          <ListingItem
-            key={listing.id}
-            id={listing.id}
-            title={listing.title}
-            price={listing.price}
-            imageUrls={listing.imageUrls}
-          />
-        ))}
-      </div>
-    </div>
+    <ListingsContainer title="Filtered Listings" listings={listings}>
+      {(listing) => (
+        <ListingItem key={listing.id} {...listing} />
```

Comment:
if this logic is being used as children for all ListingsContainer, I would recommend just putting this logic in ListingContainer itself